metadata_scylla.go

```diff
@@ -237,23 +271,25 @@ func columnKindFromSchema(kind string) (ColumnKind, error) {
 	}
 }
 
+type Metadata struct {
+	tabletsMetadata  cowTabletList
+	keyspaceMetadata cowKeyspaceMetadataMap
+}
+
 // queries the cluster for schema information for a specific keyspace and for tablets
 type schemaDescriber struct {
```

Comment:
Maybe it worth to rename it to something that represents its new role, something like `metadataKeeper`, `metadataVault`, etc.. ?